jsonschema/workflow_execution.json

```diff
@@ -0,0 +1,90 @@
+{
```

Comment:
can we add comments to this? or a README to the directory perhaps?

Reply:
done